CBO	Cantini L, Bertoli G, Cava C, Dubois T, Zinovyev A, Caselle M, Castiglioni I, Barillot E, Martignetti L. Identification of microRNA clusters cooperatively acting on epithelial to mesenchymal transition in triple negative breast cancer. 2019. <i>Nucleic Acids Res.</i> [Epub ahead of print]	https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gkz016/5290485
MMN	Noronha A, Modamio J, Jarosz Y, Guerard E, Sompairac N, Preciat G, Danielsdottir AD, Krecke M, Merten D, Haraldsdottir HS, Heinken A, Heirendt L, Magnusdottir S, Ravcheev DA, Sahoo S, Gawron P, Friscioni L, Garcia B, Prendergast M, Puente A, Rodrigues M, Roy A, Rouquaya M, Wiltgen L, Zagare A, John E, Krueger M, Kuperstein I, Zinovyev A, Schneider R, Fleming RMT, Thiele I.  The Virtual Metabolic Human database: integrating human and gut microbiome metabolism with nutrition and disease. <i>Nucleic Acids Res.</i> 2018 Oct 29. doi: 10.1093/nar/gky992.	https://academic.oup.com/nar/article/47/D1/D614/5146204
MMN	Letort G, Montagud A, Stoll G, Heiland R, Barillot E, Macklin P, Zinovyev A, Calzone L. PhysiBoSS: a multi-scale agent-based modelling framework integrating physical dimension and cell signalling. <i>Bioinformatics</i>. 2018 Aug 30. doi: 10.1093/bioinformatics/bty766.	https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/bty766/5087713
MMN	Levy N, Naldi A, Hernandez C, Stoll G, Thieffry D, Zinovyev A, Calzone L, Paulev&eacute; L. Prediction of Mutations to Control Pathways Enabling Tumor Cell Invasion with the CoLoMoTo Interactive Notebook (Tutorial). 2018. <i>Front Physiol</i> 9:787.	https://www.frontiersin.org/articles/10.3389/fphys.2018.00787/full
MMN	Naldi A, Hernandez C, Levy N, Stoll G, Monteiro PT, Chaouiya C, Helikar T, Zinovyev A, Calzone L, Cohen-Boulakia S, Thieffry D, Paulev&eacute; L. The CoLoMoTo Interactive Notebook: Accessible and Reproducible Computational Analyses for Qualitative Biological Networks. 2018. <i>Front Physiol</i> 9:680.	https://www.frontiersin.org/articles/10.3389/fphys.2018.00680/full
MMN	Calzone L, Barillot E, Zinovyev A. Logical versus kinetic modeling of biological networks: applications in cancer research. 2018. <i>Current Opinion in Chemical Engineering</i> 21, 22-31.	https://www.sciencedirect.com/science/article/pii/S2211339817300941
KNF	Mazein A, Ostaszewski M, Kuperstein I, Watterson S, Le Novere N, Lefaudeux D, De Meulder B, Pellet J, Balaur I, Saqi M, Nogueira MM, He F, Parton A, Lemonnier N, Gawron P, Gebel S, Hainaut P, Ollert M, Dogrusoz U, Barillot E, Zinovyev A, Schneider R, Balling R, Auffray C. Systems medicine disease maps: community-driven comprehensive representation of disease mechanisms. 2018. <i>NPJ systems biology and applications</i> 4 (1), 21	https://www.nature.com/articles/s41540-018-0059-y
KNF	Monraz Gomez LC, Kondratova M, Ravel JM, Barillot E, Zinovyev A, Kuperstein I. Application of Atlas of Cancer Signalling Network in preclinical studies. <i>Brief Bioinformatics</i>. 2018 May 3.	https://academic.oup.com/bib/advance-article-abstract/doi/10.1093/bib/bby031/4992169?redirectedFrom=fulltext
KNF	Ostaszewski M, Gebel S, Kuperstein I, Mazein A, Zinovyev A, Dogrusoz U, Hasenauer J, Fleming RMT, Le Novere N, Gawron P, Ligon T, Niarakis A, Nickerson D, Weindl D, Balling R, Barillot E, Auffray C, Schneider R. Community-driven roadmap for integrated disease maps. <i>Brief Bioinformatics</i>. 2018 Apr 23.	https://academic.oup.com/bib/advance-article/doi/10.1093/bib/bby024/4982567
KNF	Kondratova M, Sompairac N, Barillot E, Zinovyev A, Kuperstein I. Signalling maps in cancer research: construction and data analysis. <i>Database (Oxford)</i>. 2018  Jan 1;2018.	https://academic.oup.com/database/article/doi/10.1093/database/bay036/4964960
MLN	Lages J, Shepelyansky D, Zinovyev A. Inferring hidden causal relations between pathway members using reduced Google matrix of directed biological networks. 2018. <i>PLoS One</i> 13(1):e0190812.	https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0190812
CBO	Costa A, Kieffer Y, Scholer-Dahirel A, Pelon F, Bourachot B, Cardon M, Sirven P, Magagna I, Fuhrmann L, Bernard C, Bonneau C, Kondratova M, Kuperstein I, Zinovyev A, Givel A.-M, Parrini M.-C, Soumelis V, Vincent-Salomon A, Mechta-Grigoriou F. Fibroblast heterogeneity and immunosuppressive environment in human breast cancer. 2018. <i>Cancer Cell</i> 33(3):463-479.e10.	https://www.cell.com/cancer-cell/fulltext/S1535-6108(18)30011-4
MLN	Cantini L, Calzone L, Martignetti L, Rydenfelt M, Bl&uuml;thgen N, Barillot E, Zinovyev A. Classification of gene signatures for their information value and functional redundancy. <i>NPJ Syst Biol Appl.</i> 4:2. 2017.	https://www.nature.com/articles/s41540-017-0038-8
MMN	Montagud A, Traynard P, Martignetti L, Bonnet E, Barillot E, Zinovyev A, Calzone L. Conceptual and computational framework for logical modelling of biological networks deregulated in diseases. <i>Brief Bioinform.</i>  doi: 10.1093/bib/bbx163. 2017.	https://academic.oup.com/bib/advance-article-abstract/doi/10.1093/bib/bbx163/4716963
REV	Azencott CA, Aittokallio T, Roy S; DREAM Idea Challenge Consortium, Norman T, Friend S, Stolovitzky G, Goldenberg A. The inconvenience of data of convenience: computational research beyond post-mortem analyses. <i>Nat Methods</i> 14(10):937-938. 2017.	https://www.nature.com/articles/nmeth.4457
MLN	Kairov U, Cantini L, Greco A, Molkenov A, Czerwinska U, Barillot E, Zinovyev A. Determining the optimal number of independent components for reproducible transcriptomic data analysis. <i>BMC Genomics</i> 18(1):712. 2017.	https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-017-4112-9
MLN	Le Morvan M, Zinovyev A, Vert JP. NetNorM: Capturing Cancer-relevant Information in Somatic Exome Mutation Data with Gene Networks for Cancer Stratification and Prognosis. <i>PLoS Computational Biology</i> 13(6):e1005573. 2017.	https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1005573
MMN	Dorel M, Viara E, Barillot E, Zinovyev A^ and Kuperstein I^. NaviCom: a web application to create interactive molecular network portraits using multi-level omics data. <i>Database (Oxford)</i> 1-11. 2017.	https://academic.oup.com/database/article/doi/10.1093/database/bax026/3098441
MMN	Stoll G, Caron B, Viara E, Dugourd A, Zinovyev A, Naldi A, Kroemer G, Barillot E, Calzone L. MaBoSS 2.0: an environment for stochastic Boolean modeling. <i>Bioinformatics</i>: btx123. 2017.	https://academic.oup.com/bioinformatics/article/33/14/2226/3059141
REV	Cantini L, Caselle M, Forget A, Zinovyev A, Barillot E, Martignetti L. A review of computational approaches for the detection of microRNAs involved in cancer. <i>Frontiers in Biosciences</i> 22:1774-1791. 2017.	https://www.bioscience.org/2017/v22/af/4571/fulltext.htm
MLN	Gorban A, Mirkes E, Zinovyev A. Robust principal graphs for data approximation. <i>Archives of Data Science</i> 2(1):1:16, 2017.	https://arxiv.org/abs/1603.06828
CBO	Jdey W, Thierry S, Russo C, Devun F,  Al Abo M, Noguiez-Hellin P, Sun J.-S, Barillot E, Zinovyev A, Kuperstein I, Pommier Y, Dutreix M. Drug Driven Synthetic Lethality: bypassing tumor cell genetics with a combination of AsiDNA and PARP inhibitors. 2016. <i>Clinical Cancer Research</i>, pii: clincanres.1193.2016.	http://clincancerres.aacrjournals.org/content/23/4/1001.long
MLN	Gorban AN, Mirkes EM, Zinovyev A. Piece-wise quadratic approximations of arbitrary error functions for fast and robust machine learning. <i>Neural Netw.</i> 2016, 84:28-38.	https://www.sciencedirect.com/science/article/pii/S0893608016301113
CBO	Deveau P, Barillot E, Boeva V, Zinovyev A, Bonnet E. Calculating Biological Module Enrichment or Depletion and Visualizing Data on Large-scale Molecular Maps with ACSNMineR and RNaviCell Packages. <i>R journal</i> 2016, 8:293-306.	https://journal.r-project.org/archive/2016/RJ-2016-047/index.html
MLN	Martignetti L, Calzone L, Bonnet E, Barillot E, Zinovyev A. ROMA: Representation and Quantification of Module Activity from Target Expression Data. <i>Front Genet.</i> 2016. 7:18.	https://www.frontiersin.org/articles/10.3389/fgene.2016.00018/full
REV	Kovar H, et al (46 authors). The second European interdisciplinary Ewing sarcoma research summit - A joint effort to deconstructing the multiple layers of a complex disease. 2016. <i>Oncotarget</i> 7(8):8613-8624.	http://www.oncotarget.com/index.php?journal=oncotarget&page=article&op=view&path[]=6937&pubmed-linkout=1
MMN	Cohen PAD, Martignetti L, Robine S, Barillot E, Zinovyev A<sup>^</sup>, Calzone L<sup>^</sup>. Mathematical Modelling of Molecular Pathways Enabling Tumour Cell Invasion and Migration. 2015. <i>PLoS Computational Biology</i> 11(11):e1004571.	https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1004571
MMN	Kuperstein I, Robine S, Zinovyev A. Network biology elucidates metastatic colon cancer mechanisms. 2015. <i>Cell Cycle</i> 14(14):2189-90.	https://www.tandfonline.com/doi/full/10.1080/15384101.2015.1060816?scroll=top&needAccess=true
REV	Dorel M, Barillot E, Zinovyev A, Kuperstein I. Network-based approaches for drug response prediction and targeted therapy development in cancer. 2015. <i>Biochem Biophys Res Commun.</i> 464(2):386-91.	https://www.sciencedirect.com/science/article/pii/S0006291X15301443?via%3Dihub
KNF	Kuperstein I, Bonnet E, Nguyen HA, Cohen D, Viara E, Grieco L, Fourquet S, Calzone L, Russo C, Kondratova M, Dutreix M, Barillot E, Zinovyev A. Atlas of Cancer Signalling Network: a systems biology resource for integrative analysis of cancer data with Google Maps. 2015. <i>Oncogenesis</i> 4:e160.	https://www.nature.com/articles/oncsis201519
MMN	Remy E, Rebouissou S, Chaouiya C, Zinovyev A, Radvanyi F, Calzone L. A Modeling Approach to Explain Mutually Exclusive and Co-Occurring Genetic Alterations in Bladder Tumorigenesis. 2015. <i>Cancer Res.</i> 75(19):4042-52.	http://cancerres.aacrjournals.org/content/75/19/4042
REV	Zinovyev A. Overcoming Complexity of Biological Systems: from Data Analysis to Mathematical Modeling. 2015. <i>Mathematical Modeling of Natural Phenomena</i> 10(3), 186-205.	https://www.mmnp-journal.org/articles/mmnp/abs/2015/03/mmnp201510p186/mmnp201510p186.html
MLN	Czerwinska U, Calzone L, Barillot E, Zinovyev A. DeDaL: Cytoscape 3 app for producing and morphing data-driven and structure-driven network layouts. 2015. <i>BMC Syst Biol.</i> 14;9:46.	https://bmcsystbiol.biomedcentral.com/articles/10.1186/s12918-015-0189-4
CBO	Martignetti L, Tesson B, Almeida A, Zinovyev A, Tucker GC, Dubois T, Barillot E. Detection of miRNA regulatory effect on triple negative breast cancer transcriptome. <i>BMC Genomics</i> 2015;16:S4.	https://bmcgenomics.biomedcentral.com/articles/10.1186/1471-2164-16-S6-S4
MMN	Calzone L, Barillot E, Zinovyev A. Predicting genetic interactions from Boolean models of biological networks. 2015. <i>Integr Biol (Camb)</i> 7(8), 921–929.	https://academic.oup.com/ib/article/7/8/921/5199157
MLN	Bonnet E, Viara E, Kuperstein I, Calzone L, Cohen DP, Barillot E, Zinovyev A. NaviCell Web Service for network-based data visualization. 2015. <i>Nucleic Acids Res.</i> 43(W1):W560–W565	https://academic.oup.com/nar/article-lookup/doi/10.1093/nar/gkv450
REV	Kuperstein I, Grieco L, Cohen DP, Thieffry D, Zinovyev A<sup>^</sup>, Barillot E<sup>^</sup>. The shortest path is not the one you know: application of biological network resources in precision oncology research. 2015. <i>Mutagenesis</i> 30(2):191-204	https://academic.oup.com/mutage/article/30/2/191/1358326
MLN	Biton A, Bernard-Pierrot I, Lou Y, Krucker C, Chapeaublanc E, Rubio Perez C, Lopez Bigas N, Kamoun A, Neuzillet Y, Gestraud P, Grieco G, Rebouissou S, de Reynies A, Benhamou S, Lebret T, Southgate J, Barillot E, Allory Y, Zinovyev A.^, Radvanyi F^. Independent component analysis uncovers the landscape of the bladder tumor transcriptome and reveals insights into luminal and basal subtypes. 2014. Cell Reports 9(4), 1235-1245	https://www.cell.com/cell-reports/fulltext/S2211-1247(14)00904-8
BIO	Chanrion M, Kuperstein I, Barriere C, El Marjou F, Cohen D, Vignjevic D, Stimmer L, Paul-Gilloteaux P, Bieche I, Tavares Sdos R, Boccia GF, Cacheux W, Meseure D, Fre S, Martignetti L, Legoix-Ne P, Girard E, Fetler L, Barillot E, Louvard D, Zinovyev A<sup>^</sup>, Robine S.<sup>^</sup>. Concomitant Notch activation and p53 deletion trigger epithelial-to-mesenchymal transition and metastasis in mouse gut. 2014. <i>Nature Communications</i>, 5:5005	https://www.nature.com/articles/ncomms6005
REV	Calzone L, Kuperstein I, Cohen D, Grieco L, Bonnet E, Servant N, Hupe P, Zinovyev A<sup>^</sup>, Barillot E.<sup>^</sup>. Biological network modelling and precision medicine in oncology [in French]. 2014. <i>Bull Cancer.</i> 101(S1):18-21	https://www.researchgate.net/publication/263430999_Biological_network_modelling_and_precision_medicine_in_oncology
MMN	Kuperstein I, Cohen DP, Pook S, Viara E, Calzone L, Barillot E, Zinovyev A. NaviCell: a web-based environment for navigation, curation and maintenance of large molecular interaction maps. 2013. <i>BMC Syst Biol</i> 7(1):100	https://bmcsystbiol.biomedcentral.com/articles/10.1186/1752-0509-7-100
MMN	Stoll G, Surdez D, Tirode F, Laud K, Barillot E, Zinovyev A<sup>^</sup>, Delattre O<sup>^</sup>. Systems biology of Ewing sarcoma: a network model of EWS-FLI1 effect on proliferation and apoptosis. 2013. <i>Nucleic Acids Res.</i>, 41(19):8853-71	https://bmcsystbiol.biomedcentral.com/articles/10.1186/1752-0509-7-100
MMN	Vera-Licona P, Bonnet E, Barillot E, Zinovyev A. OCSANA: Optimal Combinations of  Interventions from Network Analysis. 2013. <i>Bioinformatics</i> 15:29: 1571-1573	https://academic.oup.com/bioinformatics/article/29/12/1571/293094
MLN	Mirkes EM, Zinovyev A, Gorban AN. Geometrical Complexity of Data Approximators. 2013. <i>Advances in Computational Intelligence</i> 7902:500-508.	https://link.springer.com/chapter/10.1007/978-3-642-38679-4_50
MMN	Zinovyev A, Kuperstein I, Barillot E, Heyer W-D. Synthetic Lethality between Gene Defects Affecting a Single Non-essential Molecular Pathway with Reversible Steps. 2013. <i>PLoS Comput Biol</i> 9(4): e1003016	https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1003016
MMN	Bonnet E, Calzone L, Rovera D, Stoll G, Barillot E, Zinovyev A. BiNoM 2.0, a Cytoscape plugin for accessing and analyzing pathways using standard systems biology formats. 2013. <i>BMC Syst Biol.</i>  7(1):18	https://bmcsystbiol.biomedcentral.com/articles/10.1186/1752-0509-7-18
MMN	Kutumova E, Zinovyev A, Sharipov R, Kolpakov F. Model composition through model reduction: a combined model of CD95 and NF-kappaB signaling pathways. 2013. <i>BMC Syst Biol.</i>  7(1):13	https://bmcsystbiol.biomedcentral.com/articles/10.1186/1752-0509-7-13
MLN	Zinovyev A, Kairov U,  Karpenyuk T, Ramanculov E. Blind Source Separation Methods For Deconvolution Of Complex Signals In Cancer Biology. 2013. <i>Biochemical and Biophysical Research Communications</i> 430(3), 1182-1187	https://www.sciencedirect.com/science/article/pii/S0006291X12023741
MLN	Zinovyev A, Mirkes E. Data complexity measured by principal graphs. 2013. <i>Computers and Mathematics with Applications</i> 65:1471-1482	https://www.sciencedirect.com/science/article/pii/S0898122112007055
MMN	Kutumova E.O, Zinovyev A, Sharipov R.N, Kolpakov F.A. [Application of model reduction methods for constructing a composite model of apoptosis signaling]. 2012. <i>Math. Biol. i Bioinformatika</i> 7(2):572-588 (In Russian)	https://www.researchgate.net/publication/286505437_The_application_of_model_reduction_methods_to_the_construction_of_the_composite_model_of_apoptotic_pathways_In_Russian
MLN	Kairov U, Karpenyuk T, Ramanculov E, Zinovyev A. Network analysis of gene lists for finding reproducible prognostic breast cancer gene signatures. 2012. <i>Bioinformation</i> 18(6):773-776	https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3449386/
MLN	Martignetti L, Laud-Duval K, Tirode F, Pierron G, Reynaud S, Barillot E, Delattre O, Zinovyev A. Antagonism Pattern Detection between MicroRNA and Target Expression in Ewing's Sarcoma. 2012. <i>PLoS One</i> 7(7):e41770	https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0041770
REV	Radulescu O, Gorban AN, Zinovyev A and Noel V. Reduction of dynamical biochemical reactions networks in computational biology. 2012. <i>Frontiers in Genetics</i> 3: 00131	https://www.frontiersin.org/articles/10.3389/fgene.2012.00131/full
MMN	Morozova N<sup>*</sup>, Zinovyev A<sup>*</sup>, Nonne N, Pritchard L-L, Gorban AN, Harel-Bellan A. Kinetic signatures of microRNA modes of action. 2012. <i>RNA</i> 18(9): 032284	https://rnajournal.cshlp.org/content/18/9/1635.long
CBO	Martignetti L, Zinovyev A, Barillot E. Identification of shortened 3prime untranslated regions from expression arrays. 2012. <i>Journal of Bioinformatics and Computational Biology</i> 10:1241001	https://www.worldscientific.com/doi/abs/10.1142/S0219720012410016
CBO	Pinna G, Zinovyev A, Araujo N, Morozova N, Harel-Bellan A. Analysis of the Growth Control Network Specific for Human Lung Adenocarcinoma Cells. 2012. <i>Mathematical Modelling of Natural Phenomena</i> 7(01):337-368	https://www.mmnp-journal.org/articles/mmnp/abs/2012/01/mmnp201271p337/mmnp201271p337.html
MLN	Boeva V, Zinovyev A, Bleakley K, Vert JP, Janoueix-Lerosey I, Delattre O, Barillot E. Control-free calling of copy number alterations in deep-sequencing data using GC-content normalization. 2011. <i>Bioinformatics</i> 27(2):268-269	https://academic.oup.com/bioinformatics/article/27/2/268/285534
MLN	Gorban A.N, Zinovyev A. Principal manifolds and graphs in practice: from molecular biology to dynamical systems. 2010. <i>Int J Neural Syst</i> 20(3):219-32	https://www.worldscientific.com/doi/abs/10.1142/S0129065710002383
MMN	Calzone L, Tournier L, Fourquet S, Thieffry D, Zhivotovsky B, Barillot E, Zinovyev A. Mathematical modelling of cell-fate decision in response to death receptor engagement. 2010. <i>PLoS Comput Biol</i> 5;6(3):e1000702	https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1000702
MMN	Zinovyev A, Morozova N, Nonne N, Barillot E, Harel-Bellan A, Gorban AN. Dynamical modeling of microRNA action on the protein translation process. 2010. <i>BMC Systems Biology</i> 4:13	https://bmcsystbiol.biomedcentral.com/articles/10.1186/1752-0509-4-13
MMN	Gorban AN, Radulescu O, Zinovyev AY. Asymptotology of chemical reaction networks. 2010. <i>Chemical Engineering Science</i> 65, 2310-2324	https://www.sciencedirect.com/science/article/pii/S0009250909005995
REV	Barillot E, Calzone L, Zinovyev A. Biologie des Systemes du Cancer. 2009. <i>Med Sci (Paris)</i>. 2009 Jun-Jul;25(6-7):601-7	https://www.medecinesciences.org/en/articles/medsci/full_html/2009/08/medsci2009256-7p601/medsci2009256-7p601.html
CBO	Guillon N, Tirode F, Boeva V, Zinovyev A, Barillot E, Delattre O. The oncogenic EWS-FLI1 protein binds in vivo GGAA microsatellite sequences with potential transcriptional activation function. 2009. <i>PLoS ONE</i> 4(3):e4932	https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0004932
REV	Aebersold R et al. Report on EU-USA Workshop: How Systems Biology Can Advance Cancer Research. 2009. <i>Molecular Oncology</i> 3(1): 9-17	https://febs.onlinelibrary.wiley.com/doi/full/10.1016/j.molonc.2008.11.003
MMN	Radulescu O, Gorban A, Zinovyev A, Lilienbaum A. Robust simplifications of multiscale biochemical networks. 2008. <i>BMC Systems Biology</i> 2:86	https://bmcsystbiol.biomedcentral.com/articles/10.1186/1752-0509-2-86
MMN	Zinovyev A, Viara E, Calzone L, Barillot E. BiNoM: a Cytoscape plugin for using and analyzing biological networks. 2008. <i>Bioinformatics</i> 24(6):876-877	https://academic.oup.com/bioinformatics/article/24/6/876/192449
CBO	Ahnert SE, Fink TMA and Zinovyev A. How much non-coding DNA do eukaryotes require?. 2008. <i>Journal of Theoretical Biology</i> 252(4):587-592	https://www.sciencedirect.com/science/article/pii/S002251930800057X?via%3Dihub
KNF	Calzone L.<sup>*</sup>, Gelay A.<sup>*</sup>, Zinovyev A.<sup>*+</sup>, Radvanyi F, Barillot E. A comprehensive modular map of molecular interactions in RB/E2F pathway. 2008. <i>Molecular Systems Biology</i> 4:174 doi:10.1038/msb.2008.7	https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2290939/
MLN	Rapaport F, Zinovyev A, Dutreix M, Barillot E, Vert J.-P. Classification of microarray data using gene networks. 2007. <i>BMC Bioinformatics</i> 8:35	https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-8-35
CBO	Gorban A, Zinovyev A. The Mystery of Two Straight Lines in Bacterial Genome Statistics. 2007. <i>Bulletin of Mathematical Biology</i> 69: 2429-2442	https://link.springer.com/article/10.1007%2Fs11538-007-9229-6
MLN	Gorban A, Sumner N, Zinovyev A. Topological grammars for data approximation. 2007. <i>Applied Mathematics Letters</i> 20(4), 382-386	https://www.sciencedirect.com/science/article/pii/S0893965906001856
MMN	Gorban A, Karlin I, Zinovyev A. Invariant grids: method of complexity reduction in reaction networks. 2005. <i>ComPlexUs</i> 2004-05;2:110-127	https://pdfs.semanticscholar.org/1e20/144f07c071ac4672b8438fe6dbddab9dfad0.pdf
MLN	Gorban A, Zinovyev A. Elastic Principal Graphs and Manifolds and their Practical Applications. 2005. <i>Computing</i> 75, 359 -379	https://link.springer.com/article/10.1007/s00607-005-0122-6
CBO	Gorban A, Popova T, Zinovyev A. Codon usage trajectories and 7-cluster structure of 143 complete bacterial genomic sequences. 2005. <i>Physica A</i> 353, 365-387	https://www.sciencedirect.com/science/article/pii/S0378437105000828
CBO	Gorban A, Popova T, Zinovyev A. Four basic symmetry types in the universal 7-cluster structure of microbial genomic sequences. 2005. <i>In Silico Biology</i> 5, 0025	http://www.bioinfo.de/isb/2005/05/0025/main.html
CBO	Carbone A, Kepes F, Zinovyev A. Codon bias signatures, organisation of microorganisms in codon space and lifestyle. 2005. <i>Mol.Biol.Evol.</i> 22(3):547-561	https://academic.oup.com/mbe/article/22/3/547/1075922
MMN	Gorban A, Karlin I, Zinovyev A. Constructive Methods of Invariant Manifolds for Kinetic Problems. 2004. <i>Physics Reports</i> 396, 197-403	https://www.sciencedirect.com/science/article/abs/pii/S0370157304001334
MMN	Gorban A, Karlin I, Zinovyev A. Invariant grids for reaction kinetics. 2004. <i>Physica A</i>, V.333, pp.106-154	https://www.sciencedirect.com/science/article/pii/S0378437103009701
CBO	Gorban A, Zinovyev A, Popova T. Seven clusters in genomic triplet distributions. 2003. <i>In Silico Biology</i>. V.3, 0039	https://content.iospress.com/articles/in-silico-biology/isb00110
CBO	Carbone A, Zinovyev A, Kepes F. Codon Adaptation Index as a measure of dominating codon bias. 2003. <i>Bioinformatics</i>. 19(13), 2005-2015	https://www.ncbi.nlm.nih.gov/pubmed/14594704
CBO	Zinovyev A, Gorban A.N, Popova T. Self-Organizing Approach for Automated Gene Identification. 2003. <i>Open Systems and Information Dynamics</i> 10(4), 321-333	https://link.springer.com/article/10.1023/B:OPSY.0000009554.93005.f6
MLN	Gorban AN, Zinovyev A.Yu. Pitenko A.A. Visualization of data: Method of elastic maps (in Russian). <i>Neurocomputeri</i>, 2002. N4. p.19-30.
MLN	Zinovyev AYu, Pitenko AA, Popova TG. Practical applications of the method of elastic maps (in Russian). <i>Neurocomputeri</i>, 2002. N4. p.31-39.
MLN	Gorban AN, Pitenko A.A, Zinov'ev A.Y, Wunsch D.C. Vizualization of any data using elastic map method. <i>Smart Engineering System Design</i>. 2001, V.11, p. 363-368.
MLN	Gorban AN, Zinovyev AYu. Method of Elastic Maps and its Applications in Data Visualization and Data Modeling. 2001. <i>International Journal of Computing Anticipatory Systems, CHAOS</i> 12, 353-369	http://cogprints.org/3088/
MLN	Gorban AN, Zinovyev AYu, Pitenko AA. Visualization of data using method of elastic maps (in Russian). <i>Informatsionnie technologii</i>. 'Mashinostrornie' Publ, Moscow, 2000. N6, P.26-35. 